fe/fe-core/src/main/java/org/apache/doris/cloud/datasource/CloudInternalCatalog.java

```diff
@@ -701,6 +701,32 @@ public void dropMaterializedIndex(long tableId, List<Long> indexIds, boolean dro
         }
     }
 
+    /**
+     * for cloud mode, drop rollupIndex/materializedIndexId in kv meta store
+     * @param tableId
+     * @param materializedIndexId
+     */
+    public void eraseIndexDropBackendReplicas(long tableId, List<Long> indexIdList) {
+        if (indexIdList == null || indexIdList.size() == 0) {
+            LOG.warn("indexIdList is empty");
+            return;
+        }
+        long tryTimes = 1;
+        while (true) {
```

Comment:
avoid using an infinite loop.
try hundreds of times and return with an error log.

Reply:
> avoid using an infinite loop. try hundreds of times and return with an error log.

may cause resource leak, are you sure?

Reply:
> avoid using an infinite loop. try hundreds of times and return with an error log.

resolved